alerts/lib/alerttask.py

```diff
@@ -206,7 +206,7 @@ def tagBotNotify(self, alert):
             alert['notify_mozdefbot'] = False
 
         # If an alert sets specific ircchannel, then we should probably always notify in mozdefbot
-        if 'ircchannel' in alert:
+        if 'ircchannel' in alert and ('ircchannel' != "" and 'ircchannel' != None):
```

Comment:
This should be
`if 'ircchannel' in alert and alert['ircchannel'] != '' and alert['ircchannel'] != None):`

as you are checking that key's value in the alert dict